### ISSUE TITLE
Create `interpreters.ts` with re-exports

### ISSUE THREAD

**Issue by jacob-alford** [jacob-alford/schemata-ts#116]
This ticket will re-export all `Schemable` instances for all the types: Decoder, Encoder, etc.  i.e. 

```typescript
export { Schemable as Decoder } from ‘./Decoder’
```

The idea is to have two imports:

```typescript
import * as S from ‘schemable-ts-types/schemas’
import * as I from ‘schemable-ts-types/interpreters’

const User = S.Struct({
  id: S.UUID({ version: 5 }),
  email: S.Email,
  username: S.ASCII,
  favouriteColour: S.HexColor
})

export type User = S.TypeOf<typeof User>

const decodeUser = I.interpret(I.Decoder)(User)
const guardUser = I.interpret(I.Guard)(User)
const eqUser = I.interpret(I.Eq)(User)
const taskDecoderUser = I.interpret(I.TaskDecoder)(User)
const arbitraryUser = I.interpret(I.Arbitrary)(User)
```

**Comment by skeate** [comment]
I think it makes sense to partially apply them and export that way

i.e.

`export const getDecoder = interpreter(Decoder.Schemable)`

etc

can also export `interpreter` from the same module in case people want to provide their own instances, but this could help save some boilerplate

**Comment by jacob-alford** [comment]
Sounds good to me!

**Comment by jacob-alford** [comment]
Closed with #135
